core/builtin.py

```diff
@@ -482,6 +482,15 @@ def Cd(argv, mem):
   state.SetGlobalString(mem, 'PWD', dest_dir)
   return 0
 
+def PrintDirStack(dir_stack):
+  stack_size = len(dir_stack)
```

Comment:
Can this be simplified to print(' '.join(dir_stack))  ?

Reply:
In retrospect, I think there may be some ternary operator magic I can use to consolidate this section of code. Will test and get back to you

Reply:
Well you can just use an if statement?

BTW `is not`  compares for identity, not equality.  It happens to work on small integers, but shouldn't be used on integers in general.

for example:

```
>>> 5 is (3+2)
True
>>> 50000 is (30000+20000)
False
>>> 50000 == (30000+20000)
True
```

But I think this code should be changed anyway.

Reply:
I was thinking the same thing, I threw in an `if` statement to just print the directory stack only if it's size is larger than 0. Should be in the latest commit now.